package.json

```diff
@@ -37,22 +39,25 @@
     "ember-cli-sass": "5.3.1",
     "ember-cli-shims": "1.0.2",
     "ember-cli-sri": "2.1.0",
+    "ember-cli-string-helpers": "1.4.0",
```

Comment:
I don't think this is worth installing if we're only using it for capitalize. @laurenbarker has some cool tricks up her sleeve to capitalize things with CSS